api/src/main/java/jakarta/activation/MailcapCommandMap.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1997, 2021 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1997, 2021, 2022 Oracle and/or its affiliates. All rights reserved.
```

Comment:
ouch, this is wrong, only start-end years should be included, see [project handbook](https://www.eclipse.org/projects/handbook/#ip-copyright-headers) for details

Reply:
(since I've noticed this late, I'll fix this myself...)